content/en/docs/collector/management.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Management
+weight: 23
+---
+
+Please be sure to review the following documentation as prerequisite:
+
+- [Getting Started][otel-collector-getting-started] to understand how to install
+  the OpenTelemetry collector.
+- [Configuration][otel-collector-configuration] for how to configure the
+  OpenTelemetry collector, setting up telemetry pipelines.
+
+This document describes how you can manage your OpenTelemetry collector
+deployment at scale.
+
+## Basics
+
+Telemetry collection at scale requires a structured approach to manage agents.
+Typical agent management tasks include:
+
+1. Querying the agent information and configuration. The agent information can
+   include its version, operating system related information, or capabilities.
+   The configuration of the agent refers to its telemetry collection setup, for
+   example, the OpenTelemetry collector
+   [configuration][otel-collector-configuration].
+1. Upgrading/downgrading agents and management of agent-specific packages,
+   including the base agent functionality and plugins.
+1. Applying new configurations to agents. This might be required because of
+   changes in the environment or due to policy changes.
+1. Health and performance monitoring of the agents, typically CPU and memory
+   usage and also agent-specific metrics, for example, the rate of processing or
+   backpressure-related information.
+1. Connection management between a control plane and the agent such as handling
+   of TLS certificates (revocation and rotation).
+
+Not every use case requires support for all of the above agent management tasks.
+In the context of OpenTelemetry task _4. Health and performance monitoring_ is
+ideally done using OpenTelemetry.
+
+## OpAMP
+
+Observability vendors and cloud providers offer proprietary solutions for agent
+management. In the open source observability space, there is an emerging
+standard that you can use for agent management: Open Agent Management Protocol
+(OpAMP).
+
+The [OpAMP specification][opamp-spec] (status: Beta) defines how to manage a
```

Comment:
You could consider removing the status from this page. It may become outdated when the status changes. The status is anyway visible on the spec page.